src/components/App/App.js

```diff
@@ -84,6 +84,7 @@ class App extends React.Component {
             isFetching: auth && auth.get('isFetching'),
             siteId: this.props.config.getIn(["backend", "site_domain"]),
             base_url: this.props.config.getIn(["backend", "base_url"], null),
+            authEndpoint: this.props.config.getIn(["backend", "auth_endpoint"]),
```

Comment:
I wonder if we should make backends get the value themselves, instead of passing them all in here. Maybe a 2.0 thing?

Reply:
2.0, but yeah that definitely makes sense. @Benaiah what do you think?